python/kfserving/kfserving/storage.py

```diff
@@ -10,7 +11,7 @@ class Storage(object):
     @staticmethod
```

Comment:
thanks @Tomcli - any chance local prefix can start with file:// ? So then scheme becomes like this

_GCS_PREFIX = "gs://"
_S3_PREFIX = "s3://"
_LOCAL_PREFIX = "file://"

Reply:
Sure I can do that, but this will probably break the backwards compatibility of any code that's depending on this.

Reply:
This is the beauty of v0.0. I don't see any issue with this. Make sure you update docs accordingly.

Reply:
Thinking a little more deeply on this, it's somewhat awkward to call this server from the command line with a local path.

I'm fine supporting the file:// prefix, but we should probably also fall back to os.path.exists(uri)

Reply:
> Thinking a little more deeply on this, it's somewhat awkward to call this server from the command line with a local path.
> 

Well - that`s the most common dev/test scenario. 

> I'm fine supporting the file:// prefix, but we should probably also fall back to os.path.exists(uri)

Yes, I agree - thanks

Reply:
@Tomcli while you are at it, it may make sense to take a stab at S3 and GCS download implementation?

Reply:
Essentially an implementation for these methods

Reply:
@animeshsingh Sure, I will do it in a separate PR since it might take me a while.

Reply:
Done